app/components/auk/search-results-list/item.js

```diff
@@ -0,0 +1,40 @@
+import Component from '@glimmer/component';
+import { computed } from '@ember/object';
+import { assert } from '@ember/debug';
+import { isPresent } from '@ember/utils';
+
+/**
+ * Search results list item. Takes most arguments that LinkTo takes for creating a search results list element that links somewhere
+ *
+ * @argument {String} class: Sets the css class of the wrapping <li>
+ * @argument {String} skin
+ * @argument {String} href: Set an (external) href link instead of composing one with LinkTo
+ */
+export default class Item extends Component {
+  // Workaround for linkTo not accepting @model and @models parameter, regardless if one is null
+  // https://github.com/emberjs/ember.js/issues/18265
+  @computed('args.{model,models}')
```

Comment:
Is die `@computed` hier expliciet met een reden gezet? Lijkt mij overbodig in een native class component waarbij de `args` sowieso `tracked` zijn. Anders misschien best documenteren dat die noodzakelijk is zodat die in de toekomst niet per ongeluk verwijderd wordt.

Reply:
Ik meen mij te herinneren dat dit hier moest (en dat @ValenberghsSven dit stuk code ook even bekeken had), maar 100% zeker ben ik er niet van. Heb je liever dat ik dit even opnieuw bekijk?

Moment van toevoegen en korte bespreking daar rond kan je trouwens terug vinden vanaf: https://github.com/kanselarij-vlaanderen/kaleidos-frontend/pull/887#issuecomment-849044713

PS: stuk code zit ook binnen `app/components/auk/button-link.js`, `app/components/auk/dropdown/item.js` & `app/components/auk/dropdown/item.js`.

Reply:
Goh, die workaround makes sense.

Je zou die ook anders kunnen aanpakken, door enkel een `@models` te aanvaarden in `<Auk::SearchResultsList::Item`
Dan volstaat het om het geval van publication-case-search `array` toe te voegen aan een singular model:

```
      <Auk::SearchResultsList::Item @route="publications.publication.documents" @models={{array result.id}} @skin="multiline">
```

Reply:
Dat documenteren we dan wel best, `Takes most arguments that LinkTo takes` is dan niet meer waar

Reply:
@ValenberghsSven De flexibiliteit hier rond (`@model` & `@models`) lijkt me op zich wel een voordeel, dus ik denk ik dit zo laat.

Ivm `@computed`: daar zonet even over opgezocht (wat altijd interessant is) en voor collega's die daar meer over willen weten: https://guides.emberjs.com/release/upgrading/current-edition/tracked-properties/. Bedankt om dit te vermelden, @erikap! Ik pas dit direct even hier aan (een noteer om dat elders ook aan te passen).